streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorNodeTest.java

```diff
@@ -118,24 +118,27 @@ public void testMetrics() {
         assertNotNull(metrics.getSensor("name-mock.0_0." + throughputOperation));
 
         for (final String opName : latencyOperations) {
-            StreamsTestUtils.getMetricByNameFilterByTags(metrics.metrics(), "mock.0_0." + opName + "-latency-avg", groupName, metricTags);
```

Comment:
L116 and 118 also have changes that should be reverted.

Reply:
hmm, I'm not sure about that.  Those lines are getting the sensor by name which still uses the combination of `thread.name() + taskID + operationName` or am I missing something?

Reply:
oh! yeah, you're right. Sorry!